sfge/src/main/java/com/salesforce/graph/ops/GraphUtil.java

```diff
@@ -137,13 +134,25 @@ public static void loadSourceFolders(GraphTraversalSource g, List<String> source
             comps.addAll(buildFolderComps(sourceFolder));
         }
 
-        // Verify all TopLevelWrappers have unique names
-        final TreeMap<String, Util.CompilationDescriptor> uniqueNames = CollectionUtil.newTreeMap();
+        // Verify TopLevelWrappers have appropriately unique names
+        final TreeMap<String, Util.CompilationDescriptor> uniqueClassNames =
```

Comment:
`uniqueClassNames` is a misleading name for a map that holds more than just classes.

Reply:
I changed it to `uniqueClassEnumInterfaceNames`. What do you think? The only other option I came up with was `uniqueObjectNames` but that is not as clear and might be confusing (is a trigger an object?).